lib/SPIRV/libSPIRV/spirv.hpp

```diff
@@ -171,6 +171,7 @@ enum ExecutionMode {
     ExecutionModeMaxWorkDimINTEL = 5894,
     ExecutionModeNoGlobalOffsetINTEL = 5895,
     ExecutionModeNumSIMDWorkitemsINTEL = 5896,
+    ExecutionModeVectorComputeFastCompositeKernelINTEL = 8088,
```

Comment:
We have recently found that there is a bug in the spec with token assignments: https://github.com/intel/llvm/pull/1612#discussion_r535084124, so this should be `6068`

Reply:
Sure, I 'll update all my pull requests